packages/kuma-gui/src/app/application/services/data-source/index.ts

```diff
@@ -140,30 +84,55 @@ export const create: Creator = (src, router) => {
       size: parseInt(queryParams.get('size') || '0'),
       page: parseInt(queryParams.get('page') || '0'),
       search: queryParams.get('search') || '',
-      cacheControl: ['no-store', 'no-cache'].reduce((prev, item) => queryParams.has(item) ? item : prev, queryParams.get('cacheControl') ?? ''),
+      cacheControl: [
+        'no-store',
+        'no-cache',
+        'immutable',
```

Comment:
an `immutable` `cacheControl` means we will only ever request the datasource once throughout the entire lifetime of the application. If you request it again, it will always use the value in the cache